examples/run_topoflow_glacier.py

```diff
@@ -37,10 +57,48 @@ def run_topoflow_glacier():
 
         model.update()
 
+        # Similiar output saving to:
+        # https://github.com/NGWPC/lstm/blob/341be45ed854442459ad2f4ed05532b5eb5406fe/lstm/run_lstm_bmi.py#L52C1-L54C27 see here for output variable
+        dest_array = np.zeros(1)
+        model.get_value("snowpack__melt_volume_flux", dest_array)
```

Comment:
This is perfect for now. We can send all these to a separate function in the future.